core/templates/dev/head/pages/exploration_editor/ExplorationSaveService.js

```diff
@@ -93,6 +94,17 @@ oppia.factory('ExplorationSaveService', [
             };
             $scope.explorationId = (
               ExplorationContextService.getExplorationId());
+            $scope.explorationLink = $window.location.protocol + '//' +
+            $window.location.host + '/explore/' + $scope.explorationId;
+            $scope.selectText = function($event) {
```

Comment:
Suggest using `evt` instead of $event (and maybe consider changing the embed modal code to do the same). In general we only use $... for Angular standard libraries.

Reply:
Done.